core/src/main/java/org/frankframework/util/CredentialFactory.java

```diff
@@ -45,10 +46,13 @@ public CredentialFactory(String alias, String defaultUsername, String defaultPas
 	}
 
 	public CredentialFactory(String alias, Supplier<String> defaultUsernameSupplier, Supplier<String> defaultPasswordSupplier) {
-		credentials = org.frankframework.credentialprovider.CredentialFactory.getCredentials(alias, defaultUsernameSupplier, defaultPasswordSupplier);
+		if (alias == null) {
+			credentials = new Credentials(alias, defaultUsernameSupplier, defaultPasswordSupplier);
```

Comment:
Ja dit is nu wat voorheen op CredentialFactory (in cf module) op regel 130 stond!